app/routes/questionnaire.py

```diff
@@ -121,8 +121,11 @@ def get_questionnaire(schema, questionnaire_store):
     language_code = get_session_store().session_data.language_code
 
     hub = HubContext(
+        language_code,
         questionnaire_store.progress_store,
         questionnaire_store.list_store,
+        questionnaire_store.answer_store,
```

Comment:
Prob not as part of this, feel like hub should just be passed questionnaire store.

Reply:
I actually started passing SummaryContext the questionnaire store for a while, but it became painful since it's used by view submission to generate the context (with no real questionnaire store).